src/transport_ofi.c

```diff
@@ -980,6 +981,38 @@ int query_for_fabric(struct fabric_info *info)
     return ret;
 }
 
+static int shmem_transport_ofi_target_ep_init() {
+    int ret = 0;
+
+    struct fabric_info* info = &shmem_transport_ofi_info;
+    info->p_info->ep_attr->tx_ctx_cnt = FI_SHARED_CONTEXT;
+    info->p_info->caps = FI_RMA | FI_ATOMICS;
```

Comment:
Do you need to also ```| FI_REMOTE_READ | FI_REMOTE_WRITE```?  Otherwise, I think we automatically get all of ```FI_REMOTE_READ/WRITE | FI_READ/WRITE```.

Reply:
Oh yeah, I forgot we don't want `FI_READ/WRITE`.